docs/learn/learn-vault-guides.md

```diff
@@ -0,0 +1,148 @@
+and type the following:
+
+`cargo run load-metadata -d -u wss://statemine.api.onfinality.io/public-ws`
+
+where `wss://statemine.api.onfinality.io/public-ws` is the OnFinality RPC endpoint for Statemine.
+This will create the file `sign_me_load_metadata_statemineV9370` under the `files/in_progress`
+folder.
+
+### Generating Signature
+
+Start the terminal within the `files/in_progress` folder and type the following:
+
+`cat sign_me_load_metadata_statemineV9370 | subkey sign --suri "YOUR SEED PHRASE"`
+
+where `"YOUR SEED PHRASE"` is the seed phrase of the account you used to sign the chain
+specification. Running the code above will return the signature below:
+
+`0xde1ad7aeb252acb3cf42a522dcc8dc3f317a49be2ed636836dd6df8f7e47135f2c712480055822eba87e9ea5ac7d3bba96045992ae795856fdf4eea09a411f85`
+
+### Signing Metadata
+
+Now, go back to the `/generate_message` folder and type the following:
+
+`cargo run --release make --goal qr --crypto sr25519 --msg load-metadata --payload sign_me_load_metadata_statemineV9370 --verifier-hex PUBLIC KEY --signature-hex SIGNATURE`
+
+where `PUBLIC KEY` is the public key of the account with seed `"YOUR SEED PHRASE"`, and `SIGNATURE`
+is the signature generated in the previous step. Running the code above will create the file
+`load_metadata_statemineV9370` under the `files/completed` folder.
+
+## Add Chain & Update Metadata
+
+You can open `add_specs_statemine-sr25519` on your browser (just drag the file on an open tab). This
+is a .png file containing the QR code to add statemine chain specification into the Vault App. You
+can do the same with the `load_metadata_statemineV9370`. This is a .apng file containing the QR code
+fountain to do the metadata update for statemine.
+
+## Metadata Portal
+
+### Modify `config` File
+
+Alternatively, you can add the chain specification and the metadata QR code fountain in a metadata
+portal. Briefly, fork the
+[Parity's Metadata Portal GitHub repository](https://github.com/paritytech/metadata-portal). You can
+modify the following fields of the `config.toml` file:
+
+- `name`: your name / institution
+- `public_key`: the public key of the account you use to sign the chain spec and the metadata.
+- then at the bottom of the file add the following information:
+
+```
+[[chains]]
+name = "statemine"
+rpc_endpoint = "wss://statemine.api.onfinality.io/public-ws"
+color = "#f27230"
+
+[chains.github_release]
+owner = "paritytech"
+repo = "statemint"
+genesis_hash = "0x48239ef607d7928874027a43a67689209727dfb3d3dc5e5b03a39bdc2eda771a"
+```
+
+For reach additional chain you need to add the respective information. Information about the genesis
+hash can be found on the Polkadot-JS UI > connect to the relevant chain > Developer > Chain State..
+
+### Rename Chain's Files
+
+Rename the singed chain spec and metadata files as follow:
+
+`add_specs_statemine-sr25519` --> `statemine_specs.png`
+
+`load_metadata_statemineV9370`--> `statemine_metadata_9370.apng`
```

Comment:
Is the file extension .png or .apng?

Reply:
.png for the chain spec Qr code and .apng for the fountain